handlers/hormuud/handler.go

```diff
@@ -79,7 +79,7 @@ type mtPayload struct {
 func (h *handler) Send(ctx context.Context, msg courier.MsgOut, res *courier.SendResult, clog *courier.ChannelLog) error {
 	token, err := h.FetchToken(ctx, msg.Channel(), msg, clog)
 	if err != nil {
-		return errors.Wrapf(err, "unable to fetch token")
+		return courier.ErrFailedWithReason("", "unable to fetch token")
```

Comment:
the point of this method is to return an error with a code from the channel's side that can be used to help the user debug when viewing the channel log... what are you trying to do here?

Reply:
return an error that can be matched in the tests

Reply:
Adjust to use STD errors